mastodon_handler.py

```diff
@@ -0,0 +1,84 @@
+import requests
+import re
+from telebot.formatting import escape_markdown
+from html import unescape
+
+def html_to_markdown(text):
```

Comment:
looks like a deadcode to me? why are you submitting it?
btw are you sure there is no external python package that does it? there is no need to reinvent the wheel